app/controllers/kitchen_controller.rb

```diff
@@ -28,10 +28,15 @@ def refresh_verification_status_from_hca!
     latest_status = identity_payload["verification_status"].to_s
     return unless User::VALID_VERIFICATION_STATUSES.include?(latest_status)
 
+    ysws_eligible = identity_payload["ysws_eligible"] == true
+
     current_user.complete_tutorial_step!(:identity_verified) if latest_status == "verified"
 
-    return if current_user.verification_status.to_s == latest_status
-    current_user.update!(verification_status: latest_status)
+    updates = {}
```

Comment:
I think you can get away without doing this! ActiveRecord will track if a record is dirty or doesn't need to be committed:

```rb
current_user.verification_status = latest_status
current_user.ysws_eligible = ysws_eligible
current_user.save # only runs if the user has changed
```